core/ui/src/components/domains/DomainUsers.vue

```diff
@@ -134,6 +142,26 @@
                           :label="$t('domain_users.change_password')"
                         />
                       </cv-overflow-menu-item>
+                      <cv-overflow-menu-item
+                        v-if="policy.expiration.enforced"
+                        :disabled="
+                          loading.alterUser ||
+                          row.user.toLowerCase() === 'administrator' ||
+                          row.password_last_set < 0
+                        "
+                        @click="enableOrDisablePasswordPolicy(row)"
+                      >
+                        <NsMenuItem
+                          :icon="
+                            row.password_expiration === -1 ? Play20 : Stop20
```

Comment:
I've talked with @AmaLuci:
- Let's always use `Power20` icon here
- Let's remove `Play20` and `Stop20` imports
- Let's change "Enable" and "Disable" labels to "Enable user" and "Disable user" for clarity (in the next kebab item)